### PR TITLE
Adding Readme for autofocus feed

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23363

## Description
Adding readme for autofocus feed

### CODE REVIEW

File: Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+| insecure | Trust any certificate \(not secure\) | False |
+| proxy | Use system proxy settings | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### autofocus-get-indicators
+***
+Gets the indicators from AutoFocus.
```
Comment (ronykoz) [inline Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md:29]: need to make sure to write that it doesn't create indicators in the system

File: Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+Use the AutoFocus Feeds integration to fetch indicators from AutoFocus.
+## Configure AutoFocus Feed on Demisto
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for AutoFocus Feed.
+3. Click **Add instance** to create and configure a new integration instance.
+
```
Comment (ronykoz) [inline Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md:7]: Where are the services here?

File: Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+##### Base Command
+
+`autofocus-get-indicators`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| limit | The maximum number of indicators to return. The default value is 10. | Required |
```
Comment (Itay4) [inline Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md:39]: `limit` shouldnt be required if there is default value

File: Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md (hunk 2 above)
Comment (roysagi) [reply Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md:9]: will add an explanation in the table below - with the param

File: Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md (hunk 1 above)
Comment (roysagi) [reply Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md:39]: done

File: Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md (hunk 3 above)
Comment (roysagi) [reply Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md]: done - also fixed in the yml.

File: Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+Note: This command does not create indicators within Cortex XSOAR.
+
+##### Base Command
+
+`autofocus-get-indicators`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| limit | The maximum number of indicators to return. The default value is 10. | Optional |
```
Comment (Itay4) [inline Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md:49]: no need to add `The default value is...`, Demisto does it automatically

Comment (ronykoz) [reply Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md:49]: @roysagi lets fix this one and merge

Comment (roysagi) [reply Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md:49]: I'll leave it in the readme and drop it in the yml as it does need to appear in the documentation

**Comment by kirbles19** [comment]
@roysagi README is done, ready to publish

**Pull request merged**